src/clipper.js

```diff
@@ -191,13 +191,15 @@ export default class Clipper {
         [this._collateral, Config.vars.dai]
       ]);
     } else if (exchangeCalleeAddress === Config.vars.collateral[this._collateralName].wstETHCurveUniv3Callee) {
+      const route = this.encodeUniv3Route()
```

Comment:
I cannot reproduce now, and the error trace would indicate we have a problem with the route coming back here as that gets thrown into the abi encoder, and is the only thing that changed.  That being said, if we have some problem here, we have it in the other codepaths too.  I'm inclined to say, there is something here, but not related to this PR.